_data/navigation.yml

```diff
@@ -32,3 +32,5 @@
   url: /blog/
 - title: FAQ
   url: /docs/faq
+- title: Contribute
+  url: /docs/contributing
```

Comment:
There is an issue for detecting the currently active item in the menu.

I will take a look, I think @existentialism already mentionned it.

![i](https://cloud.githubusercontent.com/assets/1493671/21750752/cbcbb602-d5ba-11e6-86ed-36f8ab11a933.png)